src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final String LINE_SEPARATOR = "\n";
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)  {
+        StringBuilder builder = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo).append(LINE_SEPARATOR);
+
+        LocalDate from = LocalDate.now();
+        LocalDate to = LocalDate.now();
```

Comment:
The `from` and `to` LocalDate variables are initialized with `LocalDate.now()`, which is not necessary and could lead to incorrect behavior if the parsing fails. Consider initializing them to `null` or using a different approach to handle parsing errors more effectively.